src/Simulation/EntryPointDriver/Microsoft.Quantum.EntryPointDriver.csproj

```diff
@@ -26,7 +26,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Core\Microsoft.Quantum.Runtime.Core.csproj" />
```

Comment:
Why is this no longer needed when it was needed before (not sure why it was needed before)?

Reply:
It wasn't needed before, so cleaning it up makes the dependency tree easier to track.